infrastructure/scripts/create-testdata/create-environments.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+# usage
+# ./create-environments.sh [path/to/envs]
+# Note that this just creates files, it doesn't push in git
+
+FRONTEND_PORT=8081 # see docker-compose.yml
+
+testData=${1:-"./testdata_template/environments"}
```

Comment:
```suggestion
cd $(dirname $0)
testData=${1:-"./testdata_template/environments"}
```

this script depends on the current path, so we should make sure were are where we think we are: